Tools/AP_Bootloader/board_types.txt

```diff
@@ -266,11 +266,15 @@ AP_HW_VIMDRONES_MOSAIC_X5            1405
 AP_HW_VIMDRONES_MOSAIC_H             1406
 AP_HW_VIMDRONES_PERIPH               1407
 
+
 # IDs 5000-5099 reserved for Carbonix
+# IDs 5100-5199 reserved for SYPAQ Systems
 # IDs 5200-5209 reserved for Airvolute
-AP_HW_AIRVOLUTE_DCS2             5200
+AP_HW_AIRVOLUTE_DCS2                 5200
+
+# IDs 5210-5219 reserved for Aocoda-RC
+AP_HW_AOCODA-RC-H743DUAL             5210
 
-# IDs 5100-5199 reserved for SYPAQ Systems
```

Comment:
Why are you moving this?

Reply:
Numbers（board id） are not very continuous.